translate/rus.json

```diff
@@ -129,7 +129,7 @@
 "eval":"оценка",
 "evaluation":"оценка",
 "Event PGN":"Событие PGN",
-"Event Stats":"Стат. Событий",
+"Event Stats":"Стат. События",
```

Comment:
Simple ending of the word mistype. Since this is one event we are talking about, that is why we should use the word 'событие' in singular, not plural form. The Case (if you know what that means, otherwise don't care about this sentence) of the noun remains unchanged.